src/csharp/Microsoft.Spark.E2ETest/IpcTests/ML/Feature/CountVectorizerTests.cs

```diff
@@ -70,10 +70,10 @@ public void Test_CountVectorizer()
         }
         
         /// <summary>
-        /// From Spark 2.4.0 CountVectorizer also has SetMaxDF, verify that works.
+        /// Test APIs introduced since Spark 2.4.0
         /// </summary>
         [SkipIfSparkVersionIsLessThan(Versions.V2_4_0)]
-        public void CountVectorizer_MaxDF()
+        public void CountVectorizer_Since_v2_4_0()
```

Comment:
You can follow examples like the following (from DataFrameTests.cs):
```csharp
        /// <summary>
        /// Test signatures for APIs introduced in Spark 2.4.*.
        /// </summary>
        [SkipIfSparkVersionIsLessThan(Versions.V2_4_0)]
        public void TestSignaturesV2_4_X()
```

Reply:
I copied it from DataFrameTests - if it should be 2_4_0 instead of 2_4_X let me know

Reply:
You can just name it as `TestSignaturesV2_4_X`